src/types/tracker.ts

```diff
@@ -96,3 +141,71 @@ export type AnalyticsPayload = {
     sdk_token?: string
   }
 }
+
+export type AnalyticsTrackedEventNames =
+  | 'COMPLETE'
+  | 'COMPLETE_CROSS_DEVICE_MOBILE_SUCCESS'
+  | 'COUNTRY_SELECTION'
+  | 'CROSS_DEVICE_CLIENT_INTRO'
+  | 'CROSS_DEVICE_CLIENT_INTRO'
+  | 'CROSS_DEVICE_CLIENT_INTRO'
```

Comment:
Why are multiple instances of `'CROSS_DEVICE_CLIENT_INTRO'`, `'CROSS_DEVICE_LINK_METHOD_SELECTED'`, `'DOCUMENT_CAMERA_ERROR'`, `'DOCUMENT_CAPTURE'`, `'DOCUMENT_CONFIRMATION'`, `'DOCUMENT_VIDEO_CAPTURE'`, `'FACE_SELFIE_CAPTURE'`, `'FACE_VIDEO_CAPTURE'`, `'UNSUPPORTED_BROWSER'` needed for `AnalyticsTrackedEventNames` type?

Reply:
My bad, this is because I've copy pasted the new event names from the mapping. Will change it